tests/build_rpms.sh

```diff
@@ -70,12 +72,8 @@ fi
 # Prepare the RPM environment
 mkdir -p /tmp/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
 
-if [[ $BUILD_ENV == uw_build ]]; then
-    printf "%s\n" "%dist .el${OS_VERSION}" >> /etc/rpm/macros.dist
-else
-    printf "%s\n" "%dist .${BUILD_ENV}.el${OS_VERSION}" >> /etc/rpm/macros.dist
-fi
-printf "%s\n" "%${BUILD_ENV} 1" >> /etc/rpm/macros.dist
+printf "%s\n" "%dist .el${OS_VERSION}" >> /etc/rpm/macros.dist
+[[ ${BUILD_ENV} =~ ^osg ]] && printf "%s\n" "%osg 1" >> /etc/rpm/macros.dist
```

Comment:
This syntax doesn't make sense to me, I didn't think the `[[ ... ]]` was legal without an if statement? What are you trying to do here? Should include a comment.

Reply:
Yep, `[[ ... ]] && foo` is equivalent to `if [[ ... ]]; then foo; fi`

(note that due to precedence, `[[ ... ]] && foo || bar` is _not_ the same as `if [[ ... ]]; then foo; else bar; fi`, it is `if [[ ... ]]; then foo || bar; fi`)

As a side note, a regex is not necessary here... to check if `$BUILD_ENV` starts with `osg`, you can do `[[ $BUILD_ENV == osg* ]]`.

Reply:
> As a side note, a regex is not necessary here... to check if $BUILD_ENV starts with osg, you can do [[ $BUILD_ENV == osg* ]].

Done